### PR TITLE
Add Polish translation

### PR DESCRIPTION
**Pull request by piotrdrag**

### CODE REVIEW

**Comment by Hjdskes** [comment]
Thanks! Was there enough context in the .pot file for you? If there's anything missing let me know so I can add it.

File: po/pl.po [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+msgid "Button mapping"
+msgstr "Mapowanie przycisku"
+
+#. Translators: section header for assigning special functions to buttons.
+#: piper/buttondialog.py:153
+msgid "Special mapping"
+msgstr "Mapowanie specjalne"
+
+#: piper/buttondialog.py:223 piper/buttonspage.py:82
+msgid "Button {} click"
```
Comment (piotrdrag) [inline po/pl.po]: A comment explaining what’s under {} would be very useful.

File: po/pl.po [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+msgid "Disabled"
+msgstr "Wyłączone"
+
+#. Translators: the button has an unknown function.
+#: piper/buttonspage.py:92 piper/ratbagd.py:613
+msgid "Unknown"
+msgstr "Nieznane"
+
+#: piper/buttonspage.py:100
+msgid "Configure button {}"
```
Comment (piotrdrag) [inline po/pl.po]: ^ Likewise.

File: po/pl.po [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+#: piper/mouseperspective.py:88
+msgid "Buttons"
+msgstr "Przyciski"
+
+#: piper/mouseperspective.py:90
+msgid "LEDs"
+msgstr "Diody LED"
+
+#: piper/mouseperspective.py:93 piper/profilerow.py:41
+msgid "Profile {}"
```
Comment (piotrdrag) [inline po/pl.po]: And here too.

File: po/pl.po [hunk 4]
```diff
@@ -0,0 +1,441 @@
...
+#: piper/ratbagd.py:617
+msgid "Wheel Up"
+msgstr "Kółko w górę"
+
+#: piper/ratbagd.py:618
+msgid "Wheel Down"
+msgstr "Kółko w dół"
+
+#: piper/ratbagd.py:619
+msgid "Ratchet Mode"
```
Comment (piotrdrag) [inline po/pl.po]: I had to google this one. :)

File: po/pl.po [hunk 5]
```diff
@@ -0,0 +1,441 @@
...
+msgstr "Tryb zapadkowy"
+
+#: piper/ratbagd.py:620
+msgid "Resolution Cycle Up"
```
Comment (piotrdrag) [inline po/pl.po]: “Cycle” is unclear, here and everywhere else.

File: po/pl.po [hunk 6]
```diff
@@ -0,0 +1,441 @@
...
+#: piper/ratbagd.py:623
+msgid "Resolution Down"
+msgstr "Rozdzielczość w dół"
+
+#: piper/ratbagd.py:624
+msgid "Resolution Switch"
+msgstr "Przełączenie rozdzielczości"
+
+#: piper/ratbagd.py:625
+msgid "Resolution Default"
```
Comment (piotrdrag) [inline po/pl.po]: “Default Resolution” seems more natural.

File: po/pl.po [hunk 7]
```diff
@@ -0,0 +1,441 @@
...
+#: piper/ratbagd.py:628
+msgid "Profile Up"
+msgstr "Profil w górę"
+
+#: piper/ratbagd.py:629
+msgid "Profile Down"
+msgstr "Profil w dół"
+
+#: piper/ratbagd.py:630
+msgid "Second Mode"
```
Comment (piotrdrag) [inline po/pl.po]: I have no idea how to understand this, except on the most literal level.

File: po/pl.po [hunk 8]
```diff
@@ -0,0 +1,441 @@
...
+
+#: data/ui/ButtonDialog.ui:252
+msgid "Set the device's handedness."
+msgstr "Lewo- lub praworęczność urządzenia."
+
+#: data/ui/ButtonDialog.ui:270
+msgid ""
+"This allows you to swap the order of the primary mouse buttons. In left-"
+"handedness mode, the physical left mouse button generates a right mouse "
+"click and vice versa."
```
Comment (piotrdrag) [inline po/pl.po]: Something feels fishy here. Are you sure it’s consistent with the strings below?

File: po/pl.po [hunk 9]
```diff
@@ -0,0 +1,441 @@
...
+#: data/ui/ResolutionsPage.ui:174
+msgid "Add a new resolution to the profile"
+msgstr "Dodaje nową rozdzielczość do profilu"
+
+#: data/ui/WelcomePerspective.ui:20
+msgid "Welcome"
+msgstr "Witamy"
+
+#: data/ui/WelcomePerspective.ui:38 data/ui/WelcomePerspective.ui:89
+msgid "Select a device to configure it"
```
Comment (piotrdrag) [inline po/pl.po]: I would like to translate the label and the tooltip differently. Please add context to one of them:

https://wiki.gnome.org/TranslationProject/DevGuidelines/Translation%20contexts

File: po/pl.po (hunk 8 above)
Comment (Hjdskes) [inline po/pl.po]: Yes I am sure :)

File: po/pl.po (hunk 7 above)
Comment (Hjdskes) [inline po/pl.po]: To be completely honest I'm not sure either, @whot ?

File: po/pl.po (hunk 6 above)
Comment (Hjdskes) [inline po/pl.po]: Agreed, I have changed it.

File: po/pl.po (hunk 5 above)
Comment (Hjdskes) [inline po/pl.po]: Would making it "Cycle resolution up" be better?

Comment (piotrdrag) [inline po/pl.po]: How is it different from “Resolution Up”?

File: po/pl.po (hunk 4 above)
Comment (Hjdskes) [inline po/pl.po]: Hmm.. I'm not sure how to explain this in a short one-liner for translators. In Dutch I'm almost certain that we'd also just be using the english term as we generally do for such things.

File: po/pl.po (hunk 9 above)
Comment (Hjdskes) [inline po/pl.po]: Thanks, I didn't know about translation contexts yet.

File: po/pl.po (hunk 5 above)
Comment (Hjdskes) [inline po/pl.po]: It cycles, so if you get to the maximum it will go back to the minimum.

File: po/pl.po (hunk 4 above)
Comment (piotrdrag) [inline po/pl.po]: You can use more than one line. :)

File: po/pl.po (hunk 8 above)
Comment (piotrdrag) [inline po/pl.po]: Great!

File: po/pl.po (hunk 5 above)
Comment (piotrdrag) [inline po/pl.po]: I guess your suggestion would be better.

Comment (Hjdskes) [inline po/pl.po]: Ok, done. I made it "Cycle Resolution Upwards", the same for the profiles further down.

File: po/pl.po (hunk 7 above)
Comment (whot) [inline po/pl.po]: AFAIK it's like a shift key, so button X in second mode actually sends button Y events.

Comment (piotrdrag) [inline po/pl.po]: These are usually called levels, if I remember correctly.

Comment (whot) [inline po/pl.po]: Yes, levels is the XKB term for it. I think the "second mode" term comes from some hw documentation or maybe the windows software to configure it? @bentiss may remember

**Pull request merged**